internal/cmdset/cmdset_test.go

```diff
@@ -0,0 +1,194 @@
+package cmdset
+
+import (
+	"bytes"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"sigs.k8s.io/kustomize/kyaml/fieldmeta"
+	"sigs.k8s.io/kustomize/kyaml/openapi"
+)
+
+func TestSetCommand(t *testing.T) {
```

Comment:
We should add tests for how we handle incorrect input.

Reply:
Done. More tests will be added in coming PRs.